$NetBSD: patch-af,v 1.8 2016/08/09 21:46:07 jdolecek Exp $

Add a patch to use struct statvfs on NetBSD 2.0F and newer.  Fixes PR 25891
by Andreas Wiese.
Upate __NetBSD_Version__ statvfs check from 200060000 to 299000900

--- lib/ext2fs/ismounted.c.orig	2010-05-18 03:04:39.000000000 +0000
+++ lib/ext2fs/ismounted.c
@@ -198,7 +198,11 @@ static errcode_t check_mntent(const char
 static errcode_t check_getmntinfo(const char *file, int *mount_flags,
 				  char *mtpt, int mtlen)
 {
-	struct statfs *mp;
+#if defined(__NetBSD__) && __NetBSD_Version__ >= 299000900
+        struct statvfs *mp;
+#else
+        struct statfs *mp;
+#endif
         int    len, n;
         const  char   *s1;
 	char	*s2;
